Alloy/commands/compile/parsers/Alloy.Abstract.Items.js

```diff
@@ -4,7 +4,7 @@ exports.parse = function(node, state) {
 	_.extend(state, {
 		itemArrayDefinition: {
 			parents: [
-				'Ti.UI.iOS.Toolbar',
```

Comment:
Can't we just leave this line and the `Ti.UI.iOS.Toolbar.js` to make it backwards-compatible?

Reply:
@hansemannn, probably not. The implicit namespace for `Toolbar` is removed, so  `Ti.UI.iOS.Toolbar.js` is not used during compilation.